x11-packages/gnucash/build.sh

```diff
@@ -0,0 +1,17 @@
+TERMUX_PKG_HOMEPAGE="https://gnucash.org"
+TERMUX_PKG_DESCRIPTION="Personal and small-business financial-accounting software"
+# Licenses: GPL-2.0-or-later, GPL 2 or 3, public domain, special exception for
+# linking OpenSSL.
+TERMUX_PKG_LICENSE="custom"
+TERMUX_PKG_MAINTAINER="@acozzette"
+TERMUX_PKG_VERSION="5.12"
+TERMUX_PKG_SRCURL="https://github.com/Gnucash/gnucash/releases/download/5.12/gnucash-5.12.tar.bz2"
```

Comment:
The version number in the SRCURL should not be hardcoded.
```suggestion
TERMUX_PKG_SRCURL="https://github.com/Gnucash/gnucash/releases/download/${TERMUX_PKG_VERSION}/gnucash-${TERMUX_PKG_VERSION}.tar.bz2"
```